dennis/tasks/release.py

```diff
@@ -95,8 +91,7 @@ def run(self):
             release = self.github_repo.create_git_tag_and_release(
```

Comment:
To do this, simply move the "merge_back" step (which is now last), to be just before creating the tag+release